resources/js/views/global/Supercluster.vue

```diff
@@ -115,38 +128,34 @@ async function update ()
         await axios.get('clusters', {
             params: { zoom, bbox }
         })
-            .then(response =>
-            {
-                console.log('get_clusters.update', response);
+        .then(response => {
```

Comment:
No consistency between function bracelet and arrow function bracelet. Should define one style and follow it.

Reply:
For requests, I use `=> {` but for `ifs` and `else` they go on a new line. Happy to keep everything the same if it helps